develop/sdk/examples.md

```diff
@@ -40,21 +40,22 @@ command prompt:
 package main
 
 import (
-	"io"
 	"os"
 
+	"github.com/docker/docker/client"
```

Comment:
can you move this one back in the old position? These imports should be sorted alphabetically in each group 🤗